k2/csrc/cudpp/cuda_util.h

```diff
@@ -0,0 +1,52 @@
+// this file is copied/modified from
+// https://github.com/cudpp/cudpp/blob/master/src/cudpp/cuda_util.h
+#ifndef K2_CSRC_CUDPP_CUDA_UTIL_H_
+#define K2_CSRC_CUDPP_CUDA_UTIL_H_
+
+#include <stdio.h>
+#include <stdlib.h>             /* EXIT_FAILURE */
```

Comment:
seems we only use `CUDA_SAFE_CALL` in those cudpp files? Is that possible that we delete this file and use `K2_CUDA_SAFE_CALL`

Reply:
Just want to make sure we can get consistent syncronize behavior with k2, i.e. controlled by DEBUG and K2_SYNC_KERNELS

Reply:
Fixed.